tasks/EchoesToFrameConverterTask.cpp

```diff
@@ -67,6 +67,13 @@ void EchoesToFrameConverterTask::updateHook()
         addEchoesToFrame(radar_echo, yaw_correction);
     }
 
+    if (radar_echo.sweep_data.empty()) {
+        // This may seem odd at first look, because the component is port driven.
```

Comment:
It is not. As I said repeatedly, port-driven means "called at least once after a sample was received". There is NO "exactly once" guarantee. Neither in one direction (could be called more than every time there is a sample) nor in the other (could be called once for many samples).